style.css

```diff
@@ -671,3 +723,985 @@
+  }
+
+  .yellow_square img {
+    width: 102px;
+    height: 102px;
+    left: 188px;
+    top: 187px;
+    opacity: 0.5;
+  }
+
+  .about h1 {
+    position: relative;
+    width: 295px;
+    height: 110px;
+    left: 834px;
+    top: 74px;
+    font-family: "Playfair Display", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 110px;
+    line-height: 100%;
+
+    /* identical to box height, or 110px */
+
+    color: #1a2236;
+  }
+
+  .about p {
+    position: relative;
+    width: 34vw;
+    height: 84px;
+    left: 834px;
+    top: 174px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 21px;
+    color: #1a2236;
+  }
+
+  .about button {
+    position: relative;
+    box-sizing: border-box;
+    width: 254px;
+    height: 70px;
+    left: 834px;
+    top: 240px;
+    color: #fff;
+    text-align: left;
+    padding: 14px 60px;
+    background: #1a2236;
+  }
+
+  .about img {
+    position: relative;
+    width: 24px;
+    height: 24px;
+    left: 1009px;
+    top: 300px;
+  }
+
+  .four {
+    position: relative;
+    width: 62.02px;
+    height: 62.02px;
+    right: 689px;
+    top: 650px;
+    background:
+      linear-gradient(0deg, #83d7d7, #83d7d7),
+      url(/assets/fourth.svg),
+      linear-gradient(135deg, #fec55f 0%, #fcd06d 99.25%);
+    background-blend-mode: normal, overlay, normal;
+  }
+
+  .fourth_dots {
+    position: relative;
+    width: 160px;
+    height: 160px;
+    left: 480px;
+    top: 670px;
+  }
+
+  .last-part {
+    position: relative;
+    display: flex;
+    width: 100%;
+    height: 1100px;
+    left: 0;
+    top: 600px;
+    background: #f9f9fa;
+  }
+
+  .get-started h2 {
+    position: relative;
+    width: 548px;
+    height: 110px;
+    left: 446px;
+    top: 75px;
+    font-family: "Playfair Display", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 110px;
+    line-height: 100%;
+
+    /* identical to box height, or 110px */
+
+    text-align: center;
+    color: #1a2236;
+  }
+
+  .get-started img {
+    position: absolute;
+    width: 175.38px;
+    height: 175.42px;
+    right: 1176.94px;
+    top: -28px;
+    left: 0;
+  }
+
+  .get-started article {
+    position: relative;
+    width: 548px;
+    height: 63px;
+    left: 446px;
+    top: 110px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 21px;
+    letter-spacing: -0.01em;
+    color: #1a2236;
+  }
+
+  input {
+    height: 52px;
+    width: 80%;
+    display: flex;
+    color: #1a2236;
+    background-color: transparent;
+    border-radius: 4px;
+    margin-left: 18px;
+    margin-bottom: 8px;
+    z-index: 99;
+    font-size: 14px;
+    font-weight: 400;
+  }
+
+  form {
+    position: absolute;
+    display: flex;
+    flex-direction: column;
+    top: 20%;
+    left: 50%;
+    transform: translate(-50%, 50%);
+  }
```

Comment:
- [ ] Kindly make sure your form looks like the Figma design, by centering it and respecting the sizes, borders and fonts
Your design:
![Screenshot from 2023-03-04 19-19-35](https://user-images.githubusercontent.com/97834160/222920061-d6c07cee-ef80-4d2f-bd80-62ce5684608a.png)
Figma:
![Screenshot from 2023-03-04 19-24-59](https://user-images.githubusercontent.com/97834160/222920088-54d59e2c-b6cf-4d0a-ad8d-7a025b8ef960.png)